Sources/SwiftRefactor/ConvertComputedPropertyToZeroParameterFunction.swift

```diff
@@ -0,0 +1,101 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2014 - 2024 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See https://swift.org/LICENSE.txt for license information
+// See https://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+//
+//===----------------------------------------------------------------------===//
+
+#if swift(>=6)
+public import SwiftSyntax
+#else
+import SwiftSyntax
+#endif
+
+public struct ConvertComputedPropertyToZeroParameterFunction: SyntaxRefactoringProvider {
+  public static func refactor(syntax: VariableDeclSyntax, in context: Void) -> FunctionDeclSyntax? {
+    guard syntax.bindings.count == 1,
+      let binding = syntax.bindings.first,
+      let identifierPattern = binding.pattern.as(IdentifierPatternSyntax.self)
+    else { return nil }
+
+    var statements: CodeBlockItemListSyntax
+
+    guard let typeAnnotation = binding.typeAnnotation,
+      let accessorBlock = binding.accessorBlock
+    else { return nil }
+
+    switch accessorBlock.accessors {
+    case .accessors(let accessors):
+      guard accessors.count == 1, let accessor = accessors.first,
+        accessor.accessorSpecifier.tokenKind == .keyword(.get), let codeBlock = accessor.body
```

Comment:
You also need to transfer the effect specifiers from the `get` accessor to the function, eg. to be able to convert

```swift
var foo: Int {
  get async {
    await someAsyncValue()
  }
}
```

to

```swift
func foo() async -> Int {
  await someAsyncValue()
}
```

Reply:
Done